src/main/java/com/actiontech/dble/backend/mysql/nio/handler/query/impl/join/JoinHandler.java

```diff
@@ -231,6 +231,10 @@ protected void ownThreadJob(Object... objects) {
             }
             session.setHandlerEnd(this);
             nextHandler.rowEofResponse(null, isLeft, service);
+        } catch (MySQLOutPutException e) {
```

Comment:
Whether consider scenarios where functions are used after statements such as ORDER BY?

Reply:
I tried some different cases, but can't reappear problems.
Those cases all calculate fun() in MySQL.

Reply:
I tried a few different situations, but can't reappear problems.
Because in those situations, functions are calculated by MySQL, except for "join".
 Even so，add this to all appropriate handler is a good idea